### PR TITLE
Hard code authorization scope for all ADT instances.

### PR DESCRIPTION
**Pull request by azabbasi**

### CODE REVIEW

File: sdk/digitaltwins/azure-digitaltwins-core/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 ## 1.1.0-beta.1 (Unreleased)
```
Comment (drwill-ms) [inline sdk/digitaltwins/azure-digitaltwins-core/CHANGELOG.md:3]: I don't know if you plan to do this as a separate PR, but the version needs to be changed to 1.0.1 too

**Review by drwill-ms — APPROVED** [review]

File: sdk/digitaltwins/azure-digitaltwins-core/CHANGELOG.md (hunk 1 above)
Comment (azabbasi) [reply sdk/digitaltwins/azure-digitaltwins-core/CHANGELOG.md:3]: yes, I will send out a separate PR for release preparation

Comment (azabbasi) [reply sdk/digitaltwins/azure-digitaltwins-core/CHANGELOG.md]: on a second thought, I will include the release-preparation changes in the same PR

**Review by timtay-microsoft — APPROVED** [review]

**Review by alzimmermsft — APPROVED** [review]
You'll want to update the version in the `README` to be `1.0.1` as well.

https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/digitaltwins/azure-digitaltwins-core/README.md#include-the-package
